roles/aap_setup_prepare/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+# tasks file for aap_setup_prepare
+
+# we check first so that the installation can happen without root access
+- name: check if the absolutely necessary packages are installed
+  raw: "rpm -q {{ __aap_setup_prep_required_packages | join(' ') }}"  # noqa 303
```

Comment:
I am curious why do this and not just run the yum without the when?

Reply:
It wasn't working with the yum module before without ROOT access, this was the workaround. It allows the collection to be executed without root if the prereqs are there.

Reply:
but isn't root required to install AAP?

Reply:
We're just talking about the use case where you start the cluster installation from a host _outside_ of the cluster, something like a bastion host which you might have to share with others and might not have root access.

Reply:
You definitely need the use case if you create the VMs of the cluster within your automation (outside of the scope of this collection obviously, but feasible).